Lib/test/typingdata/typing.py

```diff
@@ -0,0 +1,13 @@
+# This module is for test_inspect formatannotation
```

Comment:
"typingdata" seems a bit of a misnomer since it's the `inspect` tests that use it

Reply:
If necessary, I can open another PR to refactor the "ann_modules" into "typinganndata".